font/src/directwrite/mod.rs

```diff
@@ -0,0 +1,222 @@
+// Copyright 2019 Joe Wilm, The Alacritty Project Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//! Rasterization powered by DirectWrite
+extern crate dwrote;
+use self::dwrote::{
+    FontCollection, FontStretch, FontStyle, FontWeight, GlyphOffset, GlyphRunAnalysis,
+};
```

Comment:
Using `extern crate` isn't required (and discouraged) in Rust 2018, so it should be removed.

Additionally, the `self::` should not be necessary here.

Reply:
The font crate isn't set to use Rust 2018 yet - do we want to change that as part of this PR or should it be split out?

Reply:
I wouldn't make any major changes to the font crate until we've made up our mind about fontkit, which I'd expect to happen after the 0.3 release.

Reply:
Okay, I'll leave it at 2015 then meaning we still need this.